src/Uno.Templates/content/unoapp/MyExtensionsApp.1/Properties/launchSettings.json

```diff
@@ -31,9 +31,6 @@
 //#endif
 //#if (useWinAppSdk)
 //#if (!useMsalAuthentication)
-//#if (useMobile)
-    // Note: In order to select this profile, you'll need to comment the `Packaged` profile below until this is fixed: https://aka.platform.uno/wasdk-maui-debug-profile-issue
```

Comment:
Just want to double check and confirm with @jeromelaban that this comment is indeed no longer needed